site/en/docs/extensions/migrating/rhc/index.md

```diff
@@ -0,0 +1,178 @@
+
+
+### [User Scripts API][user-scripts-api]
+
+User Scripts are small code snippets that are usually supplied by the end user, intended for User Script managers like [TamperMonkey][tampermonkey] and [Violentmonkey][violentmonkey]. It is not possible to these managers to bundle code that is written by end users, so the User Script API exposes a way to execute code provided by the end user. This _is not_ a substitute for [chrome.scripting.executeScript][executescript], or other code execution environments - end users must enable [developer mode][dev_mode] to execute anything. If the Chrome Web Store review team thinks that this is being used in a manner other than it is intended for (i.e. code provided by the end user), it may be rejected or it's listing taken down from the store.
+
+### [chrome.debugger][chrome.debugger]
+
+The chrome.debugger API gives extensions the ability to interact with the [Chrome Devtools Protocol](https://chromedevtools.github.io/devtools-protocol/). This is the same protocol that is used for Chrome's Devtools, and an [amazing number of other tools](https://github.com/ChromeDevTools/awesome-chrome-devtools). With it, an extension can request and execute remote code. Just like User Scripts, it is not a substitute for chrome.scripting, and has a much more notable end user experience. While it is being used, the user will see a warning bar at the top of the window. If the banner is closed or dismissed, the debugging session will be terminated.
+
+{% Img src="image/DXqUldooyJOUnj3qXSYLHbUgUI93/SyeEuVZINxlaVnCHqqo2.jpg", alt="screenshot of the URL bar in Chrome that has the message 'Debugger Extension started debugging this browser'", width="800", height="18" %}
+
+
+### [Sandboxed iframes][sandboxed_iframes]
+
+If you need to evaluate a string as code, and are in a DOM environment (e.g. a content script, as opposed to a background service worker), then another option would be to use a [sandboxed iframe][sandboxed_iframes]. Extensions don't support things like `eval` by default as a safety precaution - a malicious code could put user safety and security at risk. But when the code is only executed in a known safe environment, like an iframe that has been sandboxed from the rest of the web, then those risks are greatly reduced. Within this context, the Content Security Policy that blocks the usage of eval can be lifted, allowing you to run any valid javascript code.
+
+
+If you have a use case that isn't covered, feel free to reach out to the team via the [chromium-extensions][chromium-extensions] mailing list to get feedback, or open a new ticket to request guidance from [One Stop Support][oss]
+
+
+## What to do if it is just wrong
+
+The Chrome Web Store is run by humans, and as a result the reviews are not always 100% perfect. If you believe that a mistake was made in review, you can appeal the rejection via [One Stop Support][oss]
```

Comment:
```suggestion
Enforcing policies can be nuanced and review involves manual input, which means we may sometimes agree to change a review decision. If you believe that a mistake was made in review, you can appeal the rejection via [One Stop Support][oss].
```

Reply:
I realise the point you're trying to make, but I think the previous wording made it sound a little too much like we're trying to pass blame on to reviewers? It also felt a little bit to me like we were implying there isn't good quality control.

Reply:
Interesting - I read yours with the same concern, but more...clinical. Im not passionate about the matter, so acquiescing on it :)